proto/cometbft/types/v1/params.proto

```diff
@@ -107,3 +110,24 @@ message SynchronyParams {
   google.protobuf.Duration precision = 2
       [(gogoproto.stdduration) = true];
 }
+
+message FeatureParams{
+  // vote_extensions_enable_height configures the first height during which
+  // vote extensions will be enabled. During this specified height, and for all
+  // subsequent heights, precommit messages that do not contain valid extension data
+  // will be considered invalid. Prior to this height, vote extensions will not
+  // be used or accepted by validators on the network.
+  //
+  // Once enabled, vote extensions will be created by the application in ExtendVote,
+  // passed to the application for validation in VerifyVoteExtension and given
+  // to the application to use when proposing a block during PrepareProposal.
+  google.protobuf.Int64Value vote_extensions_enable_height = 1 [(gogoproto.nullable) = true];
+
+  // PBTSParams configure how CometBFT calculates block time. BFT Time is enabled by default.
```

Comment:
```suggestion
  // pbts_enable_height configures how CometBFT calculates block time. BFT Time is enabled by default.
```

Reply:
Vetoing as this is consistent with other proto comments